contents/tutorials/fake-door-test.md

```diff
@@ -0,0 +1,150 @@
+---
+title: How to run a fake door test
+date: 2023-08-09
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-15.png
+tags: ['surveys', 'actions']
+---
+
+A fake door test is when you create a "fake" UI or experience for a product or feature you are thinking of building. When users interact with it, you tell them it isn't available (yet). This show of interest helps you validate what you plan on building.
+
+This tutorial shows you how to set up a fake door test with PostHog. We build a basic app, set up a fake door test, and then implement surveys to get more feedback.
+
+> **Warning:** Fake door tests come with risks. Showing users something that doesn't exist yet could disappoint them and hurt your reputation. Make sure to be honest and transparent with users about what you are doing.
```

Comment:
I like that we've added the warning, but it comes across as very negative and makes me not want to do a fake door test. Any way we can soften the message?

Reply:
Updated to 

> **Note:** Fake door tests have risks. Showing users something non-existent could disappoint them and hurt your reputation. Make sure to be honest and transparent with users about what you are doing.